Contents/DefaultPrefs.json

```diff
@@ -37,10 +37,10 @@
     },
 
     {
-        "id": "Fuzzy",
```

Comment:
Why remove this setting?

Reply:
Cannot use it anymore..?

Reply:
We may want to ensure that we have a compatible feature, as I am sure some people use it, as it has been a requested feature, and used for some solutions.